src/utilities/postData.js

```diff
@@ -0,0 +1,56 @@
+import { rerenderList } from './renderlist.js';
+
+export const addNewReservation = (id, userName, dateStart, dateEnd, url) => {
+  fetch(url, {
+    method: 'POST',
+    body: JSON.stringify({
+      item_id: id,
+      username: userName,
+      date_start: dateStart,
+      date_end: dateEnd,
+    }),
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  })
+    .then((res) => res)
+    .then(() => rerenderList(id, `${url}?item_id=`))
+    .catch((err) => console.log('Network Error: ', err));
```

Comment:
- [ ] for javascript best practices try always to not commit `console.log`  to your repo. It’s ugly, it kills performance and it can make confidential data be visible to anyone using the browser tools to look at your website. kindly remove it and you can see more details in [here](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) 🙏